src/test/java/MainPageTest.java

```diff
@@ -0,0 +1,77 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Properties;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+
+public class MainPageTest extends BaseTest {
+    private MainPage mainPage;
+    private String baseUrl;
+    private String email;
+    private String password;
+    private String regEmail;
+    private String regPassword;
+    private String empty = "";
+    private String searchRequest;
+
+    @BeforeClass
+    public void setUp() {
+        String commonProperties = "src/main/resources/test-properties.properties";
+        Properties properties = new Properties();
```

Comment:
Please, create appropriate class utils for property reader mathod.

Reply:
Done